ChatSecure/Classes/View Controllers/OTRMessagesViewController.h

```diff
@@ -24,7 +24,7 @@
 
 @end
 
-@interface OTRMessagesViewController : JSQMessagesViewController <OTRMessagesViewControllerProtocol, UIPopoverPresentationControllerDelegate>
+@interface OTRMessagesViewController : JSQMessagesViewController <OTRMessagesViewControllerProtocol, UIPopoverPresentationControllerDelegate, OTRRoomOccupantsViewControllerDelegate>
```

Comment:
Please fix the warning here, probably by moving this to an internal class extension inside .m file. In the future you can filter warnings to 'chatsecurecore' as shown in the image below to avoid the noise from other dependencies.

<img width="318" alt="screen shot 2017-09-14 at 2 32 30 pm" src="https://user-images.githubusercontent.com/518687/30456913-59d57c04-995a-11e7-9a9a-cf7f5d5ac55b.png">